mailbox/api/src/test/java/org/apache/james/mailbox/AbstractMailboxManagerTest.java

```diff
@@ -202,7 +202,6 @@ public void user2ShouldBeAbleToCreateNestedFoldersWithoutTheirParents() throws B
         assertThat(getMailboxManager().mailboxExists(nestedFolder, session)).isTrue();
         getMailboxManager().getMailbox(MailboxPath.inbox(session), session).appendMessage(new ByteArrayInputStream("Subject: test\r\n\r\ntestmail".getBytes()), new Date(), session, false, new Flags());
     }
-    

```

Comment:
Do not change anything on AbstractMailboxManagerTest.java  class so it's reset to the master branch.